lib/java/test/org/apache/thrift/test/TestServer.java

```diff
@@ -190,7 +193,9 @@ public static void main(String [] args) {
         if (protocol_type.equals("binary")) {
         } else if (protocol_type.equals("compact")) {
         } else if (protocol_type.equals("json")) {
-        } else if (protocol_type.equals("multiplexed")) {
+        } else if (protocol_type.equals("multi")) {
```

Comment:
I don't know much about tests. But you can get crazy here if there's a lot of procotols. 
I suggest what I did. Adding a prefix and sufix. The prefix can be matched against multiplexed (multi) the suffix is the name of the protocol of the underlaying implementation. 
This convention doesn't add anything to the code. And makes you be ready for the multiplexed whether a new protocol is added with no effort. 
Convention over configuration.